test/unittests/CMakeLists.txt

```diff
@@ -342,16 +342,15 @@ ELSEIF(${DISTRO} STREQUAL "openSUSE_12.3")
   TARGET_LINK_LIBRARIES(unitTest ${STATIC_LIBS} -lmongoclient -lboost_thread-mt -lboost_filesystem-mt -lboost_system-mt -lpthread -lssl -lcrypto)
 ELSEIF(${DISTRO} MATCHES "CentOS_6.*")
   MESSAGE("unitTest: CentOS DISTRO: '${DISTRO}'")
-  TARGET_LINK_LIBRARIES(unitTest ${STATIC_LIBS} -lmongoclient
--lboost_thread-mt -lboost_filesystem-mt -lboost_system-mt -lboost_regex-mt)
+  TARGET_LINK_LIBRARIES(unitTest ${STATIC_LIBS} -lmongoclient -lboost_thread-mt -lboost_filesystem-mt -lboost_system-mt -lboost_regex-mt)
 ELSEIF(${DISTRO} MATCHES "Debian_8.2")
   # It seems that Debian 8.2 doesn't like -mt libraries for unit tests... not sure about other Debian versions
   MESSAGE("unitTest: Debian DISTRO: '${DISTRO}'")
-  TARGET_LINK_LIBRARIES(unitTest ${STATIC_LIBS} -lmongoclient -lboost_thread -lboost_filesystem -lboost_system -lboost_regex)
+  TARGET_LINK_LIBRARIES(unitTest ${STATIC_LIBS} -lmongoclient -lboost_thread -lboost_filesystem -lboost_system -lboost_regex -lgcrypt)
```

Comment:
I don't understand why -lgcrypt is needed in Debian for compiling unit test (especially when 1) other distros, e.g. CentOS not need them, 2) this library is used by HTTPS in MHD and MHD is not used in unit test) but it seems that othwersise the unitTest binary built fails...

NTC (just anoying :)